Tests/CommandsTests/TestCommandTests.swift

```diff
@@ -303,4 +303,13 @@ final class TestCommandTests: CommandsTestCase {
         }
     }
 #endif
+
+#if false
```

Comment:
Not sure there's much point to this test. We should be able to mock out the process runner and then confirm the environment is what we expect that way (when running the test command in here). In the meantime though, I'd prefer XCTSkip here rather than a `#if false`

Reply:
Yeah that's fair, I can do that.

I'm not using `XCTSkip` because it makes the rest of the code unreachable and produces a warning diagnostic.

Reply:
Fixed the test. :)

Reply:
Thanks!